manifests/t/tushev.org/AJUpdateWatcher/2.0.4.0/tushev.org.AJUpdateWatcher.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+PackageIdentifier: tushev.org.AJUpdateWatcher
+PackageName: AJUpdateWatcher
```

This must match the value shown in control panel
```suggestion
PackageName: Update Watcher for AdoptOpenJDK
```